.gitignore

```diff
@@ -27,3 +27,7 @@ dist-*/
 dist/
 stack.yaml.lock
 result*
+
+# These are touched on every :reload in GHCi.
+# https://gitlab.haskell.org/ghc/ghc/-/issues/22669
+*.o-boot
```

Comment:
Without this files being ignored [`hspec/sensei`](https://github.com/hspec/sensei) goes into a reload-loop.

With this change `sensei` can be used during development, e.g. try:

    cabal exec -- sensei -XHaskell2010 tests/Spec.hs -isrc -itests --focused-only

Or add a `.ghci` file with:

```
-- .ghci
:set -isrc
:set -itests
:set -XHaskell2010
```
so that this becomes:

    cabal exec -- sensei tests/Spec.hs --focused-only